pkg/components/controllers.go

```diff
@@ -185,12 +195,52 @@ func startIngressController(ctx context.Context, cfg *config.Config, kubeconfigP
 		}
 	}
 
+	routerEnableCompression := "false"
```

Comment:
I wonder if we should have a separate function to generate all these parameters, now that there are 20 of them.

Reply:
@pacevedom , so the idea is to create seperate function to genereate extraParams map[string] and call it generateParamsForIngress for example?

Reply:
Yes, thats what I meant. It is getting lengthy here, wdyt?

Reply:
yes it makes sense. i'll do it.

Reply:
@pacevedom , resolved, please take a look.